src/pymovements/events/ivt.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+from typing import Optional
+import numpy as np
+
+from pymovements.transforms import vnorm
+from pymovements.events import Fixation
+
+
+def ivt(
+        x: list[list[float]] | np.ndarray,
+        v: Optional[list[list[float]] | np.ndarray],
```

Comment:
since we already import annotations, you can rewrite this as None | list[........